src/pages/ProjectDetailsPage/ProjectDetailsPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { useQuery, useReactiveVar } from '@apollo/client';
+import { Button, Typography } from '@mui/material';
+import { useParams } from 'react-router-dom';
+
+import Preloader from '../../components/Preloader';
+import { authService } from '../../graphql/auth/authService';
+import { GET_PROJECT } from '../../graphql/project/query';
+import { IProject } from '../../interfaces/IProject';
+import isAdmin from '../../utils/isAdmin';
+import { InfoWrapper, PaperWrapper } from './ProjectDetailsPage.styles';
+
+interface IProjectResult {
+  project: IProject;
+}
+
+const ProjectsDetailsPage = () => {
+  const { id } = useParams();
+  const { data, loading, error } = useQuery<IProjectResult>(GET_PROJECT, {
+    variables: { id }
+  });
+
+  const user = useReactiveVar(authService.user$);
+
+  const isUserAdmin = isAdmin(user);
+
+  return (
+    <Preloader loading={loading} error={error}>
+      <PaperWrapper elevation={7}>
+        <InfoWrapper>
+          <Typography>Name: {data?.project.name || '-'}</Typography>
+          <Typography>
+            Internal name: {data?.project?.internal_name || '-'}
+          </Typography>
+          <Typography>
+            Description: {data?.project?.description || '-'}
+          </Typography>
+          <Typography>Domain: {data?.project?.domain || '-'}</Typography>
```

Comment:
Maybe, it would be good to add '-' to the separate contant, as it is used in many places